eng/pipelines/aggregate-reports.yml

```diff
@@ -96,8 +96,6 @@ stages:
               ExclusionDataBaseFileName: "JavaScriptPoliCheckExclusion"
               PublishAnalysisLogs: false
 
-          - template: /eng/common/pipelines/templates/steps/credscan.yml
```

Comment:
can we delete this file now?

Reply:
That template is in eng/common and I'm working on removing its usage across all the repos and then I'll remove it from eng/common via the tools sync process.